gearcatalog/frontend/babel.config.js

```diff
@@ -4,4 +4,8 @@ module.exports = {
     ['@babel/preset-react'],
     ['@babel/preset-typescript'],
   ],
+  plugins: [
+    // TODO: Remove after converting GearList component to functional
```

Comment:
I recommend creating a GitHub issue to track TODOs like this and then put it here like `TODO(#123): Remove X`. That way you can keep track of these TODOs and easily find all the places that need updating when you tackle that item.

Reply:
Done